editoast/editoast_schemas/src/train_schedule/path_item.rs

```diff
@@ -32,6 +33,24 @@ pub struct PathItem {
 #[serde(untagged, deny_unknown_fields)]
 pub enum PathItemLocation {
     TrackOffset(#[schema(inline)] TrackOffset),
+    OperationalPointReference(#[schema(inline)] OperationalPointReference),
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema, Hash)]
+pub struct OperationalPointReference {
+    #[serde(flatten)]
+    #[schema(inline)]
+    pub reference: OperationalPointIdentifier,
+    #[schema(inline)]
+    #[serde(default)]
+    pub track_id: Option<Identifier>,
+    #[serde(default)]
+    pub track_label: Option<String>,
```

Comment:
From my understanding, we should have a `track_id` **XOR** a `track_label`. Therefore, it's probably only one field `track_reference` which would be of type `enum TrackReference { Id(Identifier), Label(String) }`. But maybe my understanding is incorrect and I missed something?